### PR TITLE
Creating Labeler for Compute Updates

### PR DESCRIPTION
**Pull request by grizzlytheodore**
This is from the Compute AID team focusing on SDK for compute. 
This PR adds a Github Action called labeler. It will create a label 'compute' when the specification/compute folder is being updated in a PR. 

The Github action that is being added has been tested on my own repo https://github.com/grizzlytheodore/MS-HackathonMay

More information of how Github Action Labeler: 
https://github.com/actions/labeler

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I have reviewed the [documentation](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/202/Overall-Process-of-AME-Onboarding) for the workflow.
- [x] [Validation tools](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/84/Swagger-Validation-tools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/ci-fix.md)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/FAQ.md).

### ARM API Review Checklist
- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process]( https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/212/Swagger-PR-Review).

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+compute:
```
Comment (hyonholee) [inline .github/labeler.yml:1]: Azure repo has "Compute" label.  Can we change it Compute to match the existing label?

Comment (grizzlytheodore) [reply .github/labeler.yml]: label updated to computeReview

File: .github/workflows/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: Compute Labeler
```
Comment (Sandido) [inline .github/workflows/labeler.yml:1]: I think the file name should be more specific than just 'labeler' in both files. Maybe ComputeReviewLabeler.

File: .github/labeler.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+computeReview:  
+- 'specification/compute/*'
+- 'specification/compute/**/*'
```
Comment (Sandido) [inline .github/labeler.yml:3]: Why do you need both these file paths? Isn't one a subset of the other?

File: .github/workflows/labeler.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+on:
+- pull_request
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/labeler@v2
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```
Comment (Sandido) [inline .github/workflows/labeler.yml:11]: Did you specifically generate this Secret yourself, or is it a standard one already in this repo?

Comment (grizzlytheodore) [reply .github/workflows/labeler.yml]: this is Github Actions standard generation that came with the template

File: .github/workflows/labeler.yml (hunk 2 above)
Comment (grizzlytheodore) [reply .github/workflows/labeler.yml]: This has been addressed and the files have been updated

File: .github/labeler.yml (hunk 3 above)
Comment (grizzlytheodore) [reply .github/labeler.yml]: all the examples I looked at seemed to do that to ensure all the files/folders within the path are labeled https://github.com/actions/labeler for example

**Comment by ravbhatnagar** [comment]
this does not need ARM review.

**Comment by raych1** [comment]
@grizzlytheodore , I asked the questions in the email.

**Comment by Sandido** [comment]
Consider including this type of Github Action in other repositories (SDK and PS maybe).

**Review by KrisBash — APPROVED** [review]
No API changes for ARM to review

**Pull request merged**